src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,53 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGR = 18;
+    private static final int MIN_SIZE = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        isAgeValid(user);
+        isLoginValid(user);
```

Comment:
`isSmth()` use for boolean methods
```suggestion
        validateAge(user);
        validateLogin(user);
```